File: codegen/features/xsls/backendless-codegen-xsls/ios/backendless-codegen/ObjC/CRUD/CRUD/ViewControllers/ObjectsViewController.m

#import "ObjectsViewController.h"
#import "Utils.h"

@implementation ObjectsViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.navigationItem.title = [NSString stringWithFormat:@"%@ / %@", self.tableName, self.propertyName];
}

- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
    return self.objects.count;
}

- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"ObjectCell" forIndexPath:indexPath];
    cell.textLabel.text = [utils cellTitle:[self.objects objectAtIndex:indexPath.row] type:self.type];    
    return cell;
}

@end

File: codegen/features/xsls/backendless-codegen-xsls/ios/backendless-codegen/Swift/CRUD/CRUD/ViewControllers/ObjectsViewController.swift

import UIKit

class ObjectsViewController: UITableViewController {
    
    var tableName: String = ""
    var propertyName: String = ""
    var type: String = ""
    var objects = [Any]()
    
    private let utils = Utils.shared

    override func viewDidLoad() {
        super.viewDidLoad()
        navigationItem.title = "\(tableName) / \(propertyName)"
    }

    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return objects.count
    }

    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        let cell = tableView.dequeueReusableCell(withIdentifier: "ObjectCell", for: indexPath)
        cell.textLabel?.text = utils.cellTitle(value: objects[indexPath.row], type: type)
        return cell
    }
}
